examples/node-app/docker-compose.yaml

```diff
@@ -6,3 +6,11 @@ services:
       context: .
     ports:
       - "8080:8080"
+    x-k8s:
+      liveness-probe:
+        disabled: false
```

Comment:
I'm not sure you'd noticed but I changed the spec for it slightly in the related issue. Now that we support multiple types it'd make sense to reflect it in this block as well.

Reply:
this is jsut for testing, still a draft PR. I will remove this stuff from here or update it once it's supported

Reply:
this is also still the old spec :) check here: https://github.com/appvia/kev/pull/432/files#diff-ad020f19dc8b4d3bd06cec983ef7d65389a25615a114919aa08bb865b83d696cR18 <- this should be the new spec